### PR TITLE
Refactor FXIOS-6809 [v117] Remove the word browser from Firefox homepage

### PR DESCRIPTION
**Pull request by PARAIPAN9**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-6809)

## :bulb: Description

- Update the text logo on the homepage

| Dark  | Light | 
| ------------- | ------------- |
|dark|light|

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Client/Assets/Images.xcassets/fxHomeHeaderLogoText.imageset/Contents.json
Comment (lmarceau) [inline Client/Assets/Images.xcassets/fxHomeHeaderLogoText.imageset/Contents.json:4]: Don't we prefer pdf over svg? cc @thatswinnie

Comment (adudenamedruby) [reply Client/Assets/Images.xcassets/fxHomeHeaderLogoText.imageset/Contents.json:4]: I was just going to comment on this. We use PDFs, not SVGs, so we should update the asset. :)

File: Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift
Comment (adudenamedruby) [reply Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift:42]: Also, looking at this code, I'm not seeing `userDefaults` used anywhere but the declaration, so we can probably remove that as well, if it's not needed.

File: Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift
Comment (adudenamedruby) [inline Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift:41]: Also, looking at this code, I'm not seeing `userDefaults` used anywhere but the declaration, so we can probably remove that as well, if it's not needed.

File: Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift
Comment (PARAIPAN9) [reply Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift:42]: Tot avoid a new initialization of WallpaperManager every time the applyTheme method is called, but I could move it back, no problem.

**Review by adudenamedruby — APPROVED** [review]

**Comment by adudenamedruby** [comment]
@ewachowiak waiting on your approval before merging this. :)

**Comment by ewachowiak** [comment]
Yes, this looks good from a content design perspective. Thanks!

**Pull request merged**
